### PR TITLE
mopidy-subsonic: init at 1.0.0

### PR DESCRIPTION
**Pull request by jgillich**

### CODE REVIEW

**Comment by jgillich** [comment]
```
ImportError: No module named 'connection'
```
Meh, works locally.

**Comment by jgillich** [comment]
@domenkozar Would you mind taking a look? Thanks! :)

File: pkgs/applications/audio/mopidy-subsonic/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    sha256 = "0wmf0a7aynvn32pf8p02d53a8vg3l33yvwfwdy4gcx93pm3psgp2";
+  };
+
+  propagatedBuildInputs = [ mopidy pythonPackages.py-sonic pythonPackages.pykka ];
+
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    homepage = http://www.mopidy.com/;
+    description = " Mopidy backend extension for playing music from Subsonic Music Streamer";
```
Comment (cillianderoiste) [inline pkgs/applications/audio/mopidy-subsonic/default.nix]: Nitpick: please remove the space at the start here.

**Comment by jgillich** [comment]
@cillianderoiste Ops, fixed

**Comment by cillianderoiste** [comment]
@jgillich thanks! Could it be that it's not compatible with python 3.4?

**Comment by globin** [comment]
could you add mayflower@52c9a93 that fixes SSL handling in the subsonic library and test it again?

see https://github.com/crustymonkey/py-sonic/pull/13

**Comment by jgillich** [comment]
I've added the patch, thanks! So again, this builds for me locally, don't know what's happening on Travis. I'm not actually (yet) using this because of https://github.com/rattboi/mopidy-subsonic/issues/15, but it at least it didn't show any errors when I added it to mopidy, so I thought I'd submit it anyway.

**Pull request closed without merging**